server/bundler/utils.js

```diff
@@ -23,7 +31,36 @@ function pathToRegExp( path ) {
 	return new RegExp( '^' + path + '(/.*)?$' );
 }
 
+function hashFile( path ) {
+	const md5 = crypto.createHash( 'md5' );
+	let data, hash;
+
+	try {
+		data = fs.readFileSync( path );
+		md5.update( data );
+		hash = md5.digest( 'hex' );
+		hash = hash.slice( 0, HASH_LENGTH );
```

Comment:
Why are we only keeping the 10 first characters out of 32?

I suspect this would rarely happen but by doing so there is a chance that we end up using a hash for a file that is identical to the previous one.

Reply:
Same remark

Reply:
This wasn't introduced in this PR, this code just moved from a different [place](https://github.com/Automattic/wp-calypso/blob/94e818324327109f405dff72ebf802f6f06e37ab/server/pages/index.js#L64-L83).

So I prefer not change that in this PR.